### ISSUE TITLE
ImportError: DLL load failed: The specified module could not be found.

### ISSUE THREAD

**Issue by BCDav** [Microsoft/malmo#787]
I'm trying to run the python tutorials but keep getting the following message:

Traceback (most recent call last):
  File "tutorial_1.py", line 26, in <module>
    import MalmoPython
ImportError: DLL load failed: The specified module could not be found.

I downloaded the prebuilt version of malmo.

I'm working with python3.6.7 on windows 10. I manually installed the dependencies and the versions are as follows:
java: version 1.8.0_202
ffmpeg version N-92988-g32fb83e431

In addition, I attempted to use the c++ examples, but they threw an error stating that zlib.dll could not be found. 

Thanks in advance.

**Comment by AndKram** [comment]
How did you install Malmo? Do you have MalmoPython.lib and  MalmoPython.pyd in the current directory (or on PYTHONPATH)? Do you have 7zip installed? Please see https://github.com/Microsoft/malmo/blob/master/scripts/malmo_install.ps1 for all dependencies on Windows.

**Comment by BCDav** [comment]
As some already closed issues suggested, I downloaded the windows version from the following page:
https://github.com/Microsoft/malmo/releases
I also encountered another error, that MalmoPython was not being recognised, but that was fixed by pip installing malmo.

I can confirm that I have MalmoPython.lib and MalmoPython.pyd in the same directory that I am trying to run the examples from. 

I do not have 7zip installed, I instead used winrar to extract any compressed files.

**Comment by AndKram** [comment]
Sounds like you have a partial Windows install with "pip3 install malmo". The pip package needs a compatibility files to be added. Please see bottom of page: https://github.com/Microsoft/malmo/blob/master/scripts/python-wheel/README.md and copy the files from: https://github.com/Microsoft/malmo/tree/master/scripts/python-wheel/backwards-compatible-imports  into the python example dir and see if that works for you.

**Comment by BCDav** [comment]
Didn't seem to work. Silly question, but was I supposed to copy the contents of the files into the files from Python_examples dir withthe same name or create new files?

**Comment by BCDav** [comment]
Started it from the top like you said and I finally managed to get an agent to run! Thank you!

I have a question though. How do I get other missions, such as the tutorials, to run? I tried changing "from malmo.run_mission import run; run()" to malmo.tutorial_1, but it said that no such module exists. Running "python tutorial_1.py" like tutorial.pdf suggests didn't work either. Does I need to create a run() method in each agent?
